tests/NewCypress/cypress/e2e/regression/pagination.cy.ts

```diff
@@ -0,0 +1,133 @@
+import paginationPage from "../../pages/paginationPage";
+
+describe('Pagination Tests', () => {
+
```

Comment:
Missing test cases:

- first page has no previous button
- last page has no next button
- 1 page of results:
  - has no next button and no previous button and only "1" on the bar
- first page, last page and page either side of current page are shown on the bar (e.g. for [page 30 of "trust" search](https://test.find-information-academies-trusts.education.gov.uk/search?keywords=trust&pagenumber=30) you can see `1, ⋯, 29, 30, 31, ⋯ and 73` but for [page 2 of "trust" search](https://test.find-information-academies-trusts.education.gov.uk/search?keywords=trust&pagenumber=2) you can only see `1, 2, 3, ⋯ and 73`)

Reply:
The first two should already be there but I just added a test for the last scenario